src/SourceBuild/patches/arcade/0001-Use-test-prefix-on-unofficial-build-prerelease-label.patch

```diff
@@ -0,0 +1,47 @@
+     <_PreReleaseLabel>$(PreReleaseVersionLabel)</_PreReleaseLabel>
+-    <_PreReleaseLabel Condition="'$(SemanticVersioningV1)' != 'true' and '$(PreReleaseVersionIteration)' != ''">$(PreReleaseVersionLabel).$(PreReleaseVersionIteration)</_PreReleaseLabel>
++    <_PreReleaseLabel Condition="'$(SemanticVersioningV1)' != 'true' and '$(PreReleaseVersionIteration)' != ''">$(PreReleaseVersionLabel)$(_SemVerLabelSeparator)$(PreReleaseVersionIteration)</_PreReleaseLabel>
+     <_PreReleaseLabel Condition="'$(SemanticVersioningV1)' == 'true'">$(PreReleaseVersionLabel)$(PreReleaseVersionIteration)</_PreReleaseLabel>
+-    <_PreReleaseLabel Condition="'$(ContinuousIntegrationBuild)' == 'true' and '$(OfficialBuild)' != 'true'">ci</_PreReleaseLabel>
++    <_PreReleaseLabel Condition="'$(ContinuousIntegrationBuild)' == 'true' and '$(OfficialBuild)' != 'true'">test$(_SemVerLabelSeparator)ci</_PreReleaseLabel>
+     <!-- Allow a non CI build to have an official build version label when OfficialBuild is explicitly passed-in. -->
+-    <_PreReleaseLabel Condition="'$(ContinuousIntegrationBuild)' != 'true' and '$(OfficialBuild)' != 'true'">dev</_PreReleaseLabel>
++    <_PreReleaseLabel Condition="'$(ContinuousIntegrationBuild)' != 'true' and '$(OfficialBuild)' != 'true'">test$(_SemVerLabelSeparator)dev</_PreReleaseLabel>
+ 
+-    <_BuildNumberLabels Condition="'$(VersionSuffixDateStamp)' != '' and '$(SemanticVersioningV1)' != 'true'">.$(VersionSuffixDateStamp).$(VersionSuffixBuildOfTheDay)</_BuildNumberLabels>
+-    <_BuildNumberLabels Condition="'$(VersionSuffixDateStamp)' != '' and '$(SemanticVersioningV1)' == 'true'">-$(VersionSuffixDateStamp)-$(VersionSuffixBuildOfTheDayPadded)</_BuildNumberLabels>
++    <_BuildNumberLabels Condition="'$(VersionSuffixDateStamp)' != '' and '$(SemanticVersioningV1)' != 'true'">.$(VersionSuffixDateStamp)$(_SemVerLabelSeparator)$(VersionSuffixBuildOfTheDay)</_BuildNumberLabels>
++    <_BuildNumberLabels Condition="'$(VersionSuffixDateStamp)' != '' and '$(SemanticVersioningV1)' == 'true'">-$(VersionSuffixDateStamp)$(_SemVerLabelSeparator)$(VersionSuffixBuildOfTheDayPadded)</_BuildNumberLabels>
+ 
+     <!--
+       If DotNetFinalVersionKind is specified, overrides the package version produced by the build like so:
+@@ -125,8 +127,7 @@
+         "release"    1.2.3
+     -->
+     <VersionSuffix Condition="'$(DotNetFinalVersionKind)' == 'release'"/>
+-    <VersionSuffix Condition="'$(DotNetFinalVersionKind)' == 'prerelease' and '$(SemanticVersioningV1)' != 'true'">$(_PreReleaseLabel).final</VersionSuffix>
+-    <VersionSuffix Condition="'$(DotNetFinalVersionKind)' == 'prerelease' and '$(SemanticVersioningV1)' == 'true'">$(_PreReleaseLabel)-final</VersionSuffix>
++    <VersionSuffix Condition="'$(DotNetFinalVersionKind)' == 'prerelease' and '$(SemanticVersioningV1)' != 'true'">$(_PreReleaseLabel)$(_SemVerLabelSeparator)final</VersionSuffix>
```

Comment:
I think this was the idea?

```suggestion
+    <VersionSuffix Condition="'$(DotNetFinalVersionKind)' == 'prerelease'">$(_PreReleaseLabel)$(_SemVerLabelSeparator)final</VersionSuffix>
```

Reply:
Fixed with https://github.com/dotnet/sdk/pull/48327/commits/392a5e6543416f98603873410a3c30e53d4a9dca